big_tests/tests/push_integration_SUITE.erl

```diff
@@ -164,13 +172,22 @@ init_per_group(G, Config) ->
     end,
     C.
 
+end_per_group(disco, Config) ->
+    escalus:delete_users(Config),
+    Config;
```

Comment:
Just a note - there is no need to return Config in `end_per_*` callbacks - the return value is ignored.